Sources/Public/CalendarViewContent.swift

```diff
@@ -323,6 +350,7 @@ public final class CalendarViewContent {
 
   // TODO(BK): Make all item provider closures private(set) after legacy `CalendarItem` is removed.
   var monthHeaderItemModelProvider: (Month) -> InternalAnyCalendarItemModel
+  var monthFooterItemModelProvider: (Month) -> InternalAnyCalendarItemModel?
```

Comment:
small nit: can we make this entire closure optional instead of just the return value?

```suggestion
  var monthFooterItemModelProvider: ((Month) -> InternalAnyCalendarItemModel)?
```